docs/meshstack.identity-lookup.md

```diff
@@ -204,6 +233,56 @@ let example
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+To use a non-standard attribute as the meshUsers' username (Note that this cannot be used together with AzureEmailAttributeUpdate)
+
+<!--snippet:mesh.meshfed.identitylookup.azure.username-property-update#type-->
+
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let AzureUsernameAttributeUpdate =
+    {-
+      When adding/inviting a new user to a customer a custom attribute property from the users AAD
+      extension schema can be used to fill in his username.
+
+      Cannot be used together with an AzureEmailAttributeUpdate.
+
+        propertyToUse:
+            The name of the user property which is used as the username value. E.g. 'mailNickname'.
+        formatString:
+            A printf() compatible string to replace/modify the found property with. For example to add a prefix to
```

Comment:
d: Java printf?

Reply:
Updated to say Java String.format because we seem to be using that.